frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -39,22 +35,17 @@ abstract class FlutterRustBridgeExample {
 
   Future<Uint8List> handleVecU8({required Uint8List v, dynamic hint});
 
-  Future<VecOfPrimitivePack> handleVecOfPrimitive(
-      {required int n, dynamic hint});
+  Future<VecOfPrimitivePack> handleVecOfPrimitive({required int n, dynamic hint});
```

Comment:
Btw please add a test in frb_example/pure_dart/rust as well as frb_example/pure_dart/dart

Reply:
The Rust part was already there: https://github.com/surban/flutter_rust_bridge/blob/master/frb_example/pure_dart/rust/src/api.rs#L389

I've added a test in the Dart code.